roles/openshift_logging_elasticsearch/defaults/main.yml

```diff
@@ -37,6 +37,9 @@ openshift_logging_elasticsearch_storage_group: '65534'
 
 openshift_logging_es_pvc_prefix: "{{ openshift_hosted_logging_elasticsearch_pvc_prefix | default('logging-es') }}"
 
+# config the es plugin to write kibana index based on the index mode
+openshift_logging_elasticsearch_kibana_index_mode: "shared_ops"
```

Comment:
@richm @portante Do we want to change the default so ops users go to the same index or is this too much of a surprise and we should require users to opt in?